beacon_node/beacon_chain/src/data_availability_checker/processing_cache.rs

```diff
@@ -45,7 +46,7 @@ pub struct ProcessingComponents<E: EthSpec> {
     /// Blobs required for a block can only be known if we have seen the block. So `Some` here
     /// means we've seen it, a `None` means we haven't. The `kzg_commitments` value helps us figure
     /// out whether incoming blobs actually match the block.
-    pub block_commitments: Option<KzgCommitments<E>>,
+    pub block: Option<Arc<SignedBeaconBlock<E>>>,
```

Comment:
If this is now retaining full blocks, might be good to add a metric to track `processing_cache` size in case it ever leaks

Reply:
good idea!

Reply:
added a few more metrics related metrics as well https://github.com/sigp/lighthouse/pull/5138/commits/0cf408fc2747c968b99b4e9862382090f2330047